src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -172,6 +175,7 @@ public static GitHub connect() throws IOException {
      * @deprecated
      *      Use {@link #connectToEnterpriseWithOAuth(String, String, String)}
      */
+    @Deprecated
     public static GitHub connectToEnterprise(String apiUrl, String oauthAccessToken) throws IOException {
```

Comment:
Added `@Deprecated` annotation to match `@deprecated` javadoc; this is best-practice but may start causing IDE or compile warnings for existing users of the methods.

If the method is truly deprecated, this is fine.